.github/workflows/test.yml

```
@@ -41,22 +41,32 @@ jobs:
     - name: Build C++ modules
       run: |
         mkdir -p cpp/build
-        (cd cpp/build && cmake .. && make -j8)
+        pushd cpp/build
+        cmake ..
+        make -j8
+        popd
```

I think github actions start each step from the root directory, so you don't have to return at the end.
```suggestion
        cd cpp/build
        cmake ..
        make -j8
```